src/Sentry/Breadcrumb.cs

```diff
@@ -114,7 +115,7 @@ public sealed class Breadcrumb : IJsonSerializable
         }
 
         /// <inheritdoc />
-        public void WriteTo(Utf8JsonWriter writer)
+        public void WriteTo(Utf8JsonWriter writer, IDiagnosticLogger? logger)
```

Comment:
isnt this a breaking change?

Reply:
Yes. i left a comment about this at the interface change